docs/standard/linq/parse-string.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "How to parse a string"
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 81e5686c-9658-42d8-a7e3-b11be0a2c98b
+---
+
+# How to parse a string
+
+This article shows how to create an XML tree using your choice of C# or Visual Basic.
+
+## Example: Create an XML tree
+
+he following example creates an XML tree. The C# version of the example uses the `XElement.Parse` method to parse an XML string to create the tree. This technique is also available in Visual Basic, but in this example the Visual Basic version uses XML literals.
+
+```csharp
+XElement contacts = XElement.Parse(
+    @"<Contacts>
+        <Contact>
+            <Name>Patrick Hines</Name>
+            <Phone Type=""home"">206-555-0144</Phone>
+            <Phone Type=""work"">425-555-0145</Phone>
+            <Address>
+            <Street1>123 Main St</Street1>
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>10</NetWorth>
+        </Contact>
+        <Contact>
+            <Name>Gretchen Rivas</Name>
+            <Phone Type=""mobile"">206-555-0163</Phone>
+            <Address>
+            <Street1>123 Main St</Street1>
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>11</NetWorth>
+        </Contact>
+    </Contacts>");
+Console.WriteLine(contacts);
+```
+
+```vb
+Dim contacts as XElement = _
+    <Contacts>
+        <Contact>
+            <Name>Patrick Hines</Name>
+            <Phone Type="home">206-555-0144</Phone>
+            <Phone Type="work">425-555-0145</Phone>
+            <Address>
+            <Street1>123 Main St</Street1>
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>10</NetWorth>
+        </Contact>
+        <Contact>
+            <Name>Gretchen Rivas</Name>
+            <Phone Type="mobile">206-555-0163</Phone>
+            <Address>
+            <Street1>123 Main St</Street1>
+            <City>Mercer Island</City>
+            <State>WA</State>
+            <Postal>68042</Postal>
+            </Address>
+            <NetWorth>11</NetWorth>
+        </Contact>
+    </Contacts>
+```
+
+The root `Contacts` node has two `Contact` nodes. To access some specific data in your parsed XML, use the [XElement.Elements()](xref:System.Xml.Linq.XContainer.Elements) method, which in this case returns the child elements of the root `Contacts` node. The following C# example prints the first `Contact` node to the console:
+
+```csharp
```

Comment:
Can you also add the VB version?

Here's the code:

```vb
Dim contactNodes As List(Of XElement) = contacts.Elements("Contact").ToList()
Console.WriteLine(contactNodes(0))
```

Reply:
Yes, I'm adding it.